main.go

```diff
@@ -24,6 +24,7 @@ func main() {
 	listen := fs.String("listen", ":9500", "")
 	deploymentKey := fs.String("deployment-key", "deployment", "Key to use to differentiate between two different controllers.")
 	controllerName := fs.String("controller-name", "", "Name of the replication controller to update.")
+	namespace := fs.String("namespace", "", "Namespace the replicationController belongs to.")
```

Comment:
I think `Replication controller namespace.` is a bit better phrasing for the description argument.